### DISCUSSION TITLE
onn. Beta Testing Group

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#344] [Announcements]
# Upcoming onn. Support
![preview-image](https://github.com/PRProd/HA-Firemote/assets/25486599/aa321baa-84b8-4d7c-b258-2e7dd5416de5)

I'd love to have YOU as a beta tester! Follow this thread and you'll be notified right away when new versions are ready, and you'll be first in line to share your input before launch!

If you are interested in picking one up, take a look at these resources:
 - [Walmart](https://www.walmart.com/ip/onn-Google-TV-4K-Streaming-Box-New-2023-4K-UHD-resolution/2835618394?athcpid=2835618394&athpgid=AthenaItempagesubstitutions&athcgid=null&athznid=si&athieid=v0&athstid=CS020&athguid=CZaridLXbo6_jz1uxF8G5l3QrF6tVsA-8GSG&athancid=null&athena=true) - $19.88 usd as of Mar-14-2024
 - [Amazon](https://www.amazon.com/ONN-Android-Streaming-Assistant-100026240/dp/B0B75QMC7X) - $30.99 usd as of Mar-14-2024

*Tagging @Blueion76, @michaelrobgrove, & @thos-edison as known interested parties*
<br>

## To Do List
 - [x] Release first beta version for testing
 - [x] Receive and evaluate community feedback
 - [ ] Finish app list by including/testing sideloaded apps
 - [x] Publish release candidate
 - [x] Update Documentation
 - [x] Release

## Known Limitations / Issues
 - It is highly recommended to use not just the required ADB integration, but also the Android TV remote integration.  Without the remote integration, navigation is slower, and the remote does not properly display which apps are active.

**Comment by PRProd** [comment]
### Beta 1 has been released :tada:
Good news, testers! Firemote Beta Version [v3.3.0b1](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0b1) is now available to download and test! I'd love to hear from you!

**Comment by PRProd** [comment]
### Beta 2 has been released

Firemote beta version [v3.3.0b2](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0b2) is now available for testing.  Since I haven't heard anything yet from anyone, I'm going to call this version the **Release Candidate**, and merge it into the official version soon.

If you try this version, please let me know what you think - good or bad.  Feedback is always appreciated.  Thanks!

**Reply by PRProd** [reply]
Documentation Update:
 - Device documentation for one onn. device has been completed [here](https://github.com/PRProd/HA-Firemote/wiki/onn.-Device-Support-Chart), including a FULL [ADB command chart](https://github.com/PRProd/HA-Firemote/wiki/onn.-Google-TV-4K-Streaming-Box-(2023)#adb-commands)
 - The [Firemote App Launchers](https://github.com/PRProd/HA-Firemote/wiki/Firemote-App-Launchers) chart has been updated with a column for .onn apps.

**Comment by thos-edison** [comment]
Everything has been working like a charm here.

I have the onn. plugged into a video capture device that has a pass-through so I can view TV in a window on my PC, the TV, or both at the same time. Firemote saves me from having to always keep a remote at one place or the other. An added bonus is that I can turn onn. off and on via Firemote and turn the actual TV off/on using the remote - letting me move between devices at will.

I haven't tried it with my DirecTV Stream device (AT&T Osprey/Gemini) but I certainly will when I get some time.

Thanks again.
________________________________

**Comment by PRProd** [comment]
# onn. is here! :tada: 
The onn. device family is now available for everyone in the latest release, version [v3.3.0](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.0)!

I will leave this discussion group open for a while in case anybody has an issue to report or questions to ask.
